src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -51,6 +51,10 @@
     this.manifestFolder = manifestFolder;
   }
 
+  Map<Integer, Contest> getContests() {
```

Comment:
@moldover Is it intentional to have two different `getContest` functions?

Reply:
Definitely not!  Probably the second one should be named parseContestData or something like that.

Reply:
Done.